src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx

```diff
@@ -58,33 +53,127 @@ If you'll need to do a good amount of partitioning of users' access to accounts
+If you've successfully completed migration, learn [how to manage your users](#manage-users). 
 
-* **User management**: use this to view and add users, change their type (basic vs. full), change their group, and approve user upgrade requests. 
-* **Organization and access**: use this to create access grants (granting groups access to roles and accounts), and configure authentication domains (SAML SSO settings and SCIM settings, and more). 
+## Step 1: Create admins [#page1]
 
-For more about these tools, see the [new user management docs](/docs/accounts/accounts-billing/new-relic-one-user-management/introduction-managing-users/).
+![User migration page 1](./images/user-migration-page-1.png "user-migration-page-1.png")
+
+Tips: 
+* You can either a) import all current admins for your account or b) specify the admins that should have access to user management capabilities. Note that you can add more admin users and edit permissions after you complete the migration process.
+* If you've already used the wizard to set up an admin on the new user model, have the admin sign in using their new user record to access the migration tool. The user migration wizard, when completed, destroys the old user record, but if you've started the user migration process without completing it, you may have users with access to both the original and new record, as shown below: 
+	 ![A screenshot of what is shown when you have an email address associated with multiple New Relic logins](./images/login-multiple-accounts-view.png "Login UI for email addresses with multiple user records")
+    <figcaption>
+	  If a user on the new model has been created and the migration process hasn't been completed, they may have access to the original user record and the new login(s). 
+    </figcaption>
+* If you plan on migrating only a portion of your users to the new user model to start, we recommend leaving some original model admins so that you have an admin to manage the original model users. 
+
+
+## Step 2: Set up organization [#page2]
+
+![User migration page 2](./images/user-migration-page-2.png "user-migration-page-2.png")
+
+You may choose a) a guided setup that allows more configuration options, or b) an automatic setup with fewer steps. Some notes on the automatic setup:
+
+* It won't transition SSO or [AUM-implemented SCIM configurations](/docs/accounts/accounts/automated-user-management/automated-user-provisioning-single-sign). Instead, users will be provisioned manually and will log in manually with their username and password. 
+* It provides all users with access to all accounts. If you use SSO or AUM today or require some accounts to be inaccessible by certain user groups, don't use the automatic option.
+
+
+## Step 3: Name your organization [#page3]
+
+![User migration page 3](./images/user-migration-page-3.png "user-migration-page-3.png")
```

Comment:
I think you can crop out the left nav from this image

Reply:
Another no caption

Reply:
i think it's possible to do so, but I think it's helpful in letting them know 'this is part of the user migration wizard in UI'. I don't think it detracts. (I think actually the more long horizontal-wise rectangles display a bit better on page but that's a separate issue).